src/components/footer/index.js

```diff
@@ -30,84 +38,165 @@ const Paragraph = styled.p`
+  color: #fff;
+  padding: 15px;
+`;
+const FooterSocialIcon = styled.ul`
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  margin-left: 10px;
+`;
+const FooterLists = styled.li`
+  padding-left: 15px;
+
+  :first-child {
+    padding-left: 0px;
+  }
+  a {
+    color: #fff;
+  }
+`;
+const CopyRight = styled.div`
+  padding-left: 15px;
+
+  li:first-child {
+    padding-left: 6px;
+  }
+  a {
+    color: #fff;
+  }
+`;
+const FooterColumn = styled(Col)`
+  display: flex;
+  display: flex;
+  justify-content: space-between;
+`;
 class Footer extends Component {
   render() {
     return (
       <>
-        <Container>
-          <Row>
-            <Col lg="6" md="6" xs="12">
-              <Col lg="12" md="12" xs="12">
-                <FooterList>
-                  <a
-                    href="https://www.libonomy.com/"
-                    target="_blank"
-                    rel="follow">
-                    <Logoo
-                      src={Logo}
-                      alt="logo-proper-six"
-                      class="img-footer"
-                    />
-                  </a>
+        <FooterWrapper>
+          <Container>
+            <Row>
+              <Col lg="6" md="12" xs="12">
+                <Col lg="12" md="12" xs="12">
+                  <UnorderList>
+                    <Link href="#">
+                      <Logoo
+                        src={Logo}
+                        alt="logo-proper-six"
+                        class="img-footer"
+                      />
+                    </Link>
+
+                    <Paragraph>
+                      Libonomy LTD | 483 Green Lanes, N13 4BS | London United
+                      Kingdom
+                    </Paragraph>
+                    <Paragraph>info@libonomy.com</Paragraph>
+                    <Paragraph>+44 742 458 6677</Paragraph>
+                  </UnorderList>
+                </Col>
+              </Col>
+              <Col lg="2" md="4" xs="12">
+                <UnorderList>
+                  <FooterList>
+                    <Link to="#">About Us</Link>
+                  </FooterList>
+                  <FooterList>
+                    <Link to="#">Advertise</Link>
+                  </FooterList>
+                  <FooterList>
+                    <Link to="#">Terms & Conditions</Link>
+                  </FooterList>
+                </UnorderList>
+              </Col>
+              <Col lg="2" md="4" xs="12">
+                <UnorderList>
+                  <FooterList>
+                    <Link to="#">Developers API</Link>
+                  </FooterList>
+                  <FooterList>
+                    <Link href="#">Information Center</Link>
+                  </FooterList>
+                  <FooterList>
+                    <Link to="#">Newsletter</Link>
+                  </FooterList>
+                </UnorderList>
+              </Col>
+              <Col lg="2" md="4" xs="12">
+                <UnorderList>
```

Comment:
Use List as wrapper and ListItem as a child

Reply:
Wrapper is already used instead of wrapper I used List